trade_client.py

```diff
@@ -1,34 +1,43 @@
+from datetime import datetime
 from logger import logger
 
 from auth.gateio_auth import *
-from gate_api import ApiClient, Configuration, Order, SpotApi
-from gate_api.exceptions import ApiException, GateApiException
-
-from store_order import store_order
+from gate_api import ApiClient, Order, SpotApi
 
 client = load_gateio_creds('auth/auth.yml')
 spot_api = SpotApi(ApiClient(client))
-import json
 
+current_last_trade = None
 
 
-def get_last_price(base,quote, return_price_only):
+def get_last_price(base, quote, return_price_only):
     """
     Args:
     'DOT', 'USDT'
     """
-    tickers = spot_api.list_tickers(currency_pair=f'{base}_{quote}')
-    assert len(tickers) == 1
-    t = tickers[0]
+    global current_last_trade
+    trades = spot_api.list_trades(currency_pair=f'{base}_{quote}', limit=1)
+    assert len(trades) == 1
+    t = trades[0]
```

Comment:
I'd always suggest naming variables properly. Auto-complete makes sure we don't need to type them out anyway. 👍

Reply:
Fair point - I should have improved this as I was touching the code anyway